### PR TITLE
Solving #662 , Radio Button Empty Value problem

### PR DESCRIPTION
**Pull request by Jain-Sanchit**
Previously there was no check for empty value of radio button . It was being processes as there was no check for empty string .
I have applied a check condition for empty string now . If string would be empty , it will return false.
Thus this solves issue #662 .

### CODE REVIEW

**Comment by Jain-Sanchit** [comment]
@hetunandu Sir , Please have a look and let me know if something is needed to be changed.

File: app/client/src/utils/Validators.ts [hunk 1]
```diff
@@ -365,6 +365,9 @@ export const VALIDATORS: Record<ValidationType, Validator> = {
     }
     const hasOptions = _.every(parsed, (datum: { label: any; value: any }) => {
       if (_.isObject(datum)) {
+        if(_.isEmpty(datum.label) || _.isEmpty(datum.value)){
+          return false;
+        }
         return _.isString(datum.label) && _.isString(datum.value);
```
Comment (hetunandu) [inline app/client/src/utils/Validators.ts:371]: This is correct but the condition can be more readable as so

```suggestion
        return (
            _.isString(datum.label) 
            && _.isString(datum.value) 
            && !_.isEmpty(datum.label) 
            && !_.isEmpty(datum.value)
        );
```

**Review by hetunandu — CHANGES_REQUESTED** [review]

**Comment by Jain-Sanchit** [comment]
@hetunandu  Changes have been made Sir . I hope it's okay now.

**Comment by hetunandu** [comment]
Awesome! This works. Can you add a cypress test to assert this functionality? The input should have a red border now whenever any option is an empty string.

**Comment by Jain-Sanchit** [comment]
@hetunandu  I have to give it a try , I will try to implement this .

**Comment by Jain-Sanchit** [comment]
Now after updating if the value is empty , the blocks go red and shows error .
I think this was needed , please review @hetunandu 
radioError

**Review by hetunandu — APPROVED** [review]
LGTM
